dao.py

```diff
@@ -267,13 +267,30 @@ def get_tournament_by_id(self, id):
     # basically, get players who have an alias similar to the given alias
     def get_players_with_similar_alias(self, alias):
         alias_lower = alias.lower()
+
+        #here be regex dragons
+        re_test_1 = '([1-9]+.[1-9]+.)(.+)' # to match '1 1 slox'
+        re_test_2 = '(.[1-9]+.[1-9]+.)(.+)' # to match 'p1s1 slox'
+
         similar_aliases = list(set([
             alias_lower,
             alias_lower.replace(" ", ""), # remove spaces
             re.sub(special_chars, '', alias_lower), # remove special characters
             # remove everything before the last special character; hopefully removes crew/sponsor tags
-            re.split(special_chars, alias_lower)[-1].strip()
+            re.split(special_chars, alias_lower)[-1].strip(),
+            # regex nonsense to deal with pool prefixes
+            re.split(re_test_1, alias_lower)[2],
+            re.split(re_test_2, alias_lower)[2],
+            # well, we're using set, so why not
+            re.split(re_test_1, alias_lower)[2].strip(),
+            re.split(re_test_2, alias_lower)[2].strip(),
         ]))
+
+        # XXX add in all the components, sort of a last ditch hack. possibly take this out if its getting too many false positives
+        for s in re.split(" ", alias_lower):
+            if s not in similiar_aliases:
```

Comment:
it's a set, so you can just do `similar_aliases.update(re.split(" ", alias_lower))`

on the other hand, I don't think we should just add everything in `alias_lower.split()`; I think we should only include suffixes, i.e.
```python
alias_words = alias_lower.split()
similar_aliases.update([' '.join(alias_words[i:]) for i in xrange(len(alias_words))])
```